microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots.md

```diff
@@ -219,6 +219,26 @@ _Internal nodes_ are objects that are specific to V8 (the JavaScript engine in M
 To hide internal nodes from the **Retainers** section, in the **Filter edges** dropdown menu, select **Hide internal**.
 
 
+<!-- ====================================================================== -->
+## Configure the Shallow Size column to include an entire object's size
+
+By default, the **Shallow Size** column in the **Memory** tool only includes the size of the object itself. Most objects have a small shallow size because they only store their own object descriptions in memory. For example, JavaScript `Array` instances store the contents of the array in a separate memory location that's not included in the array's shallow size.
```

Comment:
```suggestion
By default, the **Shallow Size** column in the **Memory** tool only includes the size of the object itself. Most objects have a small shallow size, because they typically only store their own object descriptions in memory. For example, JavaScript `Array` instances store the contents of the array in a separate memory location that's not included in the array's shallow size.
```

Reply:
The "one place" and the "other place" are both within the JS heap in the renderer memory.